core/src/models/Source.ts

```diff
@@ -302,7 +305,10 @@ export class Source extends LoggedModel<Source> {
     return this.idIsDefault() ? ConfigWriter.generateId(this.name) : this.id;
   }
 
-  async getConfigObject() {
+  async getConfigObject(): Promise<
+    | SourceConfigurationObject
+    | [SourceConfigurationObject, ScheduleConfigurationObject]
+  > {
```

Comment:
This is one is pretty strange, but at least now the types will remind us of that!